app/models/resource.py

```diff
@@ -131,7 +131,13 @@ def print_resources():
             print resource.text_descriptors
             print resource.option_descriptors
 
-
+    @staticmethod
+    def get_resources():
+        resources = []
+        for resource in db.session.query(Resource).all():
```

Comment:
Instead of `db.session.query(Resource).all()`, can we use `Resource.query.all()` (to conform with other db calls)?